doc-content/jbpm-docs/src/main/asciidoc/Overview/processengine-overview-con.adoc

```diff
@@ -0,0 +1,47 @@
+* _Gateway_: a branching or merging node. A typical gateway evaluates an expression and, depending on the result, continues to one of several execution paths. Diamond shapes on the diagram represent gateways.
+
+When a user starts the process, a process instance is created. The process instance contains a set of data, or _context_, stored in process variables. The _state_ of a process instance includes all the context data and also the current active node (or, in some cases, several active nodes). 
+
+Some of these variables can be initialized when a user starts the process. An activity can read from process variables and write to process variables. A gateway can evaluate process variables to determine the execution path.
+
+For example, a purchase process in a shop can be a business process. The content of the user’s cart can be the initial process context. At the end of execution, the process context can contain the payment confirmation and shipment tracking details. 
+
+Optionally, you can use the BPMN data modeler in {CENTRAL} to design the model for the data in process variables.
+
+The workflow diagram is represented in code by an XML _business process definition_. The logic of events, gateways, and subprocess calls are defined within the business process definition. 
+
+Some task types (script tasks and standard decision engine rule task) are implemented in the engine. For other task types, including all custom tasks, when the task must be executed the {PROCESS_ENGINE} executes a call using the _Work Item Handler API_. COde external to the engine can implement this API, providing a flexible mechanism for implementing various tasks. 
+
+The {PROCESS_ENGINE} includes a number of predefined types of tasks. These types include a script task that runs user Java code, a service task that calls a Java method or a Web Service, a decision task that calls a decision engine service, and other custom tasks (for example, REST and database calls).
+
+Another predefined type of task is a _user task_, which includes interaction with a user. User tasks in the process can be assigned to users and groups. 
+
+The {PROCESS_ENGINE} uses the _KIE API_ to interact with other software components. You can run business processes as services on a {KIE_SERVER} and interact with them using a REST implementation of the KIE API. Alternatively, you can embed business processes in your application and interact with them using KIE API Java calls. In this case, you can run the {PROCESS_ENGINE} in any Java environment.
+
+{CENTRAL} includes a user interface for users executing human tasks and a form modeler for creating the web forms for human tasks. However, you can also implement a custom user interface that interacts with the {PROCESS_ENGINE} using the KIE API.
+
+Additional features of the {PROCESS_ENGINE} include:
```

Comment:
Same IBM suggestion to avoid incompletes before colon, so could just revise to "The {PROCESS_ENGINE} supports the following additional features:"